sdk/containerregistry/azure-containerregistry/samples/README.md

```diff
@@ -16,7 +16,8 @@ The async versions of the samples require Python 3.6 or later.
 
 |**File Name**|**Description**|
 |-------------|---------------|
-|[sample_create_client.py][create_client] and [sample_create_client_async.py][create_client_async]|Instantiate a client|Authorizing a `ContainerRegistryClient` object and `ContainerRepositoryClient` object|
+|[sample_create_client.py][create_client] and [sample_create_client_async.py][create_client_async]|Instantiate a client | Authorizing a `ContainerRegistryClient` object and `ContainerRepositoryClient` object |
```

Comment:
not sure if you want to adopt this, but in EH/SB, all the samples are listed like: [`sample.py (async version)`](https://github.com/swathipil/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/samples/README.md#azure-event-hubs-client-library-for-python-samples). May look cleaner, depending on how you want it to look.
```suggestion
|[sample_create_client.py][create_client] ([async version][create_client_async]) |Instantiate a client | Authorizing a `ContainerRegistryClient` object and `ContainerRepositoryClient` object |
```